### ISSUE TITLE
Upgrade to sendgrid v.3.2

### ISSUE THREAD

**Issue by robregonm** [bryglen/yii2-sendgrid#6]
It would be great to use the new API features (version 3.2.0).
https://github.com/sendgrid/sendgrid-php
The one I'd like to have into this extension is: Send emails using the API key.

**Comment by bryglen** [comment]
what's new in Version 3.2?

**Comment by bryglen** [comment]
I've checked the code there is some breaking changes if I updated it to 3.2. I will try to do it on a weekend

**Comment by igmilano** [comment]
Is CC working?
I can send the mail to multiple recipients but i need them in CC for forwarding.
Couldn´t found a workaround

**Comment by bryglen** [comment]
it's possible. you need to call some sendgrid methods

https://github.com/sendgrid/sendgrid-php/blob/master/lib/SendGrid/Email.php#L167
https://github.com/bryglen/yii2-sendgrid/blob/master/Message.php#L115

**Comment by igmilano** [comment]
Yes, I read the whole sendgrid documentation but I`m curently using your Yii2-sendgrid wrapper.
When I call setCcs() method, I get "Calling unknown method: bryglen\sendgrid\Message::setCcs()"

My code is

 $sendGrid = Yii::$app->sendGrid;
 $mail = $sendGrid->compose( 'new', [
                    'data'     => $data,
                    'logo'  => '../mail/images/logo.png'
] );
$mail->setFrom( [ Yii::$app->params[ "adminEmail" ] => Yii::$app->params[ "title" ] ] );
$mail->setTo("MYMAIL@gmail.com");
$mail->setCcs($emails);
$mail->setSubject( "$message" )->send( $sendGrid );

**Comment by igmilano** [comment]
For example, I tried out with your example on "how to use it" and:

$sendGrid = Yii::$app->sendGrid;
$message = $sendGrid->compose('contact/html', ['contactForm' => $form])
$message->setFrom('from@domain.com')
    ->setTo("email1@email1.com")
    ->setSubject("subject")
    ->send($sendGrid);

works perfect but

$sendGrid = Yii::$app->sendGrid;
$message = $sendGrid->compose('contact/html', ['contactForm' => $form])
$message->setFrom('from@domain.com')
    ->setTo("email1@email1.com")
    ->setCc("email2@email2.com")
    ->setSubject("subject")
    ->send($sendGrid);

dont
